src/libs/deploy.js

```diff
@@ -168,15 +169,57 @@ export const deployGoogle = async (signedApp, workspace = '/tmp/') => {
 };
 
 /**
- * Apple Store Deployment
+ * App Store Deployment
  *
- * @param {String} signedApp The name of the signed app
+ * @param {String} signedApp The name of the signed app, ONLY accepts .ipa / .pkg format
  * @param {string} [workspace='/tmp/'] The workspace to use
  * @returns The status of the deployment
  */
-// eslint-disable-next-line no-unused-vars
-export const deployAppleStore = async (signedApp, workspace = '/tmp/') => {
-  // TODO
+// eslint-disable-next-line import/prefer-default-export
+export const deployAppStore = async (signedApp, workspace = '/tmp/') => {
+  // Get app:
+  const signedAppPath = await fetchFileFromStorage(signedApp, workspace);
+  const signedAPP = require('fs').readFileSync(signedAppPath);
+
+  // Get altool access:
+  const iosKeys = ['appleAccount', 'appleKey'];
+  const iosKeyPairs = await fetchKeychainValue(iosKeys, 'altool');
+
+  try {
+    // Step 1. Use altool to validate the app:
+    // TODO: (sh) only accepting iOS apps, could provide more option for tvOS, OS X and macOS apps
+    await exec(`
+    xcrun altool --validate-app \
+    -t ios \
+    -f ${signedAPP} \
+    -u ${iosKeyPairs[iosKeys[0]]} \
```

Comment:
It would be easier to read the code if you had `const userIndex = 0` and `const passwordIndex = 1` at the top then used them to access the array.